modules/structs/user.go

```diff
@@ -43,6 +43,8 @@ type User struct {
 	Website string `json:"website"`
 	// the user's description
 	Description string `json:"description"`
+	// User visibility level option: public(0) - default, limited(1), private(2)
```

Comment:
can you change it to string?

Reply:
Done

Reply:
@sergey-dryabzhinsky thanks, but you have top propagate it to all places used :/